phoenix-core/src/main/java/org/apache/phoenix/util/PhoenixRuntime.java

```diff
@@ -632,6 +645,16 @@ public static ExecutionCommand parseArgs(String[] args) {
             return execCmd;
         }
 
+        private static String validateTableName(String tableName) {
```

Comment:
no this is to validate during upgrade only, if user is passing tablename with ":" by mistake in thought of hbase tablename.